frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -17,27 +17,31 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
   const errorMessage = (() => {
     if (!error) return undefined;
     switch (error) {
-      // Invalid interpretation
+      // Invalid ballot interpretations
       case 'invalid_test_mode':
-        return isTestMode ? 'Live Ballot detected.' : 'Test ballot detected.';
+        return isTestMode
+          ? 'Live ballot detected. Scanner is in test mode.'
+          : 'Test ballot detected. Scanner is in live mode.';
       case 'invalid_election_hash':
-        return 'Scanned ballot does not match the election this scanner is configured for.';
+        return 'The ballot does not match the election this scanner is configured for.';
       case 'invalid_precinct':
-        return 'Scanned ballot does not match the precinct this scanner is configured for.';
+        return 'The ballot does not match the precinct this scanner is configured for.';
       case 'unreadable':
-        return 'There was a problem reading this ballot. Please try again.';
       case 'unknown':
-        return undefined;
-      // Precinct scanner error
+        return 'There was a problem reading this ballot. Please scan again.';
+      // Precinct scanner errors
       case 'both_sides_have_paper':
+        return 'Scanner detected unexpected ballots in both the front and back. Remove both ballots to continue.';
```

Comment:
So this case is a bit complex:
- When a ballot is inserted in the front while there's already a ballot in the scanner, we show the scanner busy screen
- After the front ballot is removed, we'll try to continue scanning. In some cases, we can't continue, so the back ballot will be rejected and held in front, at which point this message will show.

So I might just say something like:
> Remove the ballot and try again.

Or if we really want to explain what happened, maybe something like:
> Scanning interrupted. Remove the ballot and try again.

But I don't think the voter necessarily will care or need to know in this case.